src/components/main/PublishedItems.tsx

```diff
@@ -0,0 +1,61 @@
+import { List } from 'immutable';
+
+import { Box } from '@mui/material';
+
+import { Member } from '@graasp/sdk';
+import { BUILDER } from '@graasp/translations';
+import { Loader } from '@graasp/ui';
+
+import { useBuilderTranslation } from '../../config/i18n';
+import { hooks } from '../../config/queryClient';
+import {
+  PUBLISHED_ITEMS_ERROR_ALERT_ID,
+  PUBLISHED_ITEMS_ID,
+} from '../../config/selectors';
+import ErrorAlert from '../common/ErrorAlert';
+import { useCurrentUserContext } from '../context/CurrentUserContext';
+import ItemHeader from '../item/header/ItemHeader';
+import Items from './Items';
+import Main from './Main';
+
+const PublishedItemsDisplay = ({ member }: { member: Member }): JSX.Element => {
+  const { t: translateBuilder } = useBuilderTranslation();
+  const {
+    data: sharedItems,
+    isLoading,
+    isError,
+  } = hooks.usePublishedItemsForMember(member.id);
+
+  if (isError) {
+    return <ErrorAlert id={PUBLISHED_ITEMS_ERROR_ALERT_ID} />;
+  }
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  return (
+    <Main>
```

Comment:
Having the loader be rendered above the `Main` component will introduce a flicker in the interface. I think the loader should be displayed lower, or you should wrap the Main at the level of the `PublishedItems` component. This issue is described in #589 Only the favorite items screen is implemented correctly in my memories.